src/Microsoft.AspNetCore.Server.Kestrel.Core/CoreStrings.resx

```diff
@@ -117,10 +117,193 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="BadRequest" xml:space="preserve">
+    <value>Bad request.</value>
+  </data>
+  <data name="BadRequest_BadChunkSizeData" xml:space="preserve">
+    <value>Bad chunk size data.</value>
+  </data>
+  <data name="BadRequest_BadChunkSuffix" xml:space="preserve">
+    <value>Bad chunk suffix.</value>
+  </data>
+  <data name="BadRequest_ChunkedRequestIncomplete" xml:space="preserve">
+    <value>Chunked request incomplete.</value>
+  </data>
+  <data name="BadRequest_FinalTransferCodingNotChunked" xml:space="preserve">
+    <value>Final transfer coding is not "chunked": "{detail}"</value>
```

Comment:
I updated this error message to use single quotes. Also, I didn't understand the error message when I first saw it, so I changed it to `The message body length cannot be determined because the final transfer encoding was set to '{detail}' instead of 'chunked'.`. Okay with you?